fink_science/random_forest_snia/processor.py

```diff
@@ -272,8 +272,14 @@ def extract_features_rf_snia(jd, fid, magpsf, sigmapsf, cdsxmatch, ndethist) ->
 
     return pd.Series(concatenated_features)
 
+
 @pandas_udf(DoubleType(), PandasUDFType.SCALAR)
-def rfscore_sigmoid_elasticc(midPointTai, filterName, psFlux, psFluxErr, cdsxmatch, nobs, maxduration=None, model=None) -> pd.Series:
+def rfscore_sigmoid_elasticc(midPointTai, filterName, psFlux, psFluxErr,
+                             cdsxmatch, nobs,
+                             meta=['ra', 'dec', 'hostgal_ra', 'hostgal_dec',
```

Comment:
The problem is the `meta` argument -- you cannot pass a list of names, but you need to pass the data actually. In pseudo-code this means:

```python
def rfscore_sigmoid_elasticc(..., cdsxmatch, nobs, ra, dec, hostgal, ...):
```
and then when you call the function you need to extract these columns. See e.g. what is done for CATS:

https://github.com/astrolabsoftware/fink-science/blob/896f43eb2851e7afe02d6ccebdb3fc843f916839/fink_science/cats/processor.py#L119-L122

Reply:
Thanks Julien, this now should be fixed, along with the modification the companion file
https://github.com/marcol480/fink-broker/commit/4ee0b336311ab434c5a656b31b957ad055f6b122